oe_content.tokens.inc

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Builds placeholder replacement tokens for oe_content content types.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Render\BubbleableMetadata;
+
+/**
+ * Implements hook_token_info().
+ *
+ * Provides short title token.
```

Comment:
We don't need this comment. In case later we add some other tokens, this becomes moot. Also, it's clear that the hook is for.